src/main/resources/com.atex.plugins.themes.PluginsCss.vm

```diff
@@ -1,11 +1,23 @@
 ## Render plugin contributed css
 #foreach ($webResource in $m.local.webResources.filter("type=css", "scope=external"))
-<link rel="stylesheet" type="text/css" href="$webResource.identifier" />
+  #if($string_utils.endsWith($webResource.identifier, '-print.css'))
```

Comment:
I think this would be neater if you added "mediaType" attribute to webResources in the model.

That then negates the need for multiple "if" statements and checks on the "-print.css".

The "-print.css" is business logic so should exist in the Model/Controller layer, not in the view layer.

<link rel="stylesheet" media="$webResource.mediaType" type="text/css" href="$webResource.identifier" />